app/assets/stylesheets/custom.scss

```diff
@@ -267,10 +267,45 @@ ul, menu, dir {
 //      LISTING SHOW PAGE
 //
 //    =========================*/
+.delete-listing {
+  margin-left: 20px;
+  margin-top: 15px;
+}
+
+.edit-listing-header {
+  font-size: 40px;
+}
+
+.listing-show-page {
+  margin-bottom: 20px;
+  padding: 20px;
+}
 
 .listing-header {
-  margin-top: 100px;
+  margin-top: 50px;
+  margin-left: 50px;
+}
+
+.user-show-row {
+  background: #C5EAEA;
+  margin-right: 50px;
```

Comment:
Properties should be ordered background, margin-left, margin-right, margin-top, padding